dont-worry-be-happy/dwbh-api
# Add user to group

**How to test**
1. As an admin, add an user to a group. It should success.
2. As a non-admin, add an user to a group. It should fail.
3. As an admin, add a non-existing user to a group. It should fail.
4. Add an user to a non-existing group. It should fail.
5. As an admin, add an user to a group in which the user was already a member. It should fail.
